src/confluent_kafka/schema_registry/_async/schema_registry_client.py

```diff
@@ -696,9 +697,39 @@ async def get_schema(
 
         return registered_schema.schema
 
+    async def get_schema_string(
+        self, schema_id: int, subject_name: Optional[str] = None, fmt: Optional[str] = None
+    ) -> str:
+        """
+        Fetches the schema associated with ``schema_id`` from the
+        Schema Registry. Only the unescaped schema string is returned.
+
+        Args:
+            schema_id (int): Schema id.
+            subject_name (str): Subject name the schema is registered under.
+            fmt (str): Format of the schema.
+
+        Returns:
+            str: Schema string for this version.
+
+        Raises:
+            SchemaRegistryError: if the version can't be found or is invalid.
+
+        See Also:
+            `GET Schema API Reference <https://docs.confluent.io/current/schema-registry/develop/api.html#get--schemas-ids-int-%20id-schema>`_
+        """  # noqa: E501
+
+        query = {'subject': subject_name} if subject_name is not None else None
```

Comment:
It can be slightly cleaner to do `else {}` or two lines with `query = {}` and `if subject_name:` so you don't have to do the `if query is not None` later on

Reply:
Good suggestion! Refactored